koku/koku/celery.py

```diff
@@ -85,10 +85,10 @@ def readiness_check(self):
         self._write_response(ProbeResponse(status, msg))
 
 
-def validate_cron_expression(expresssion):
+def validate_cron_expression(expresssion, default):
```

Comment:
This could be an optional parameter using the current default. Returning the literal string`default` violates the contract of this function, which is to validate a crontab or return a valid fallback value.
```suggestion
def validate_cron_expression(expresssion, default="0 4,14 * * *"):
```

Reply:
The default/fallback value should also be validated.